sonaranalyzer-dotnet/rspec/cs/S4022_c#.html

```diff
@@ -0,0 +1,41 @@
+<p>By default the storage type of an <code>enum</code> is <code>Int32</code>, in most cases it is not necessary to change this. In particular you will
+not achieve any performance gain by using a smaller data type (e.g. <code>Byte</code>) and may limit future uses.</p>
+<h2>Noncompliant Code Example</h2>
+<pre>
+using System;
+
+namespace MyLibrary
```

Comment:
Good idea. I've updated the tests and rspec, but not the resource strings - script will take care of it before release.